server/controllers/CommentController.js

```diff
@@ -240,6 +240,43 @@ class CommentController {
       comment,
     });
   }
+
+  /**
+  * @description This method deletes a comment on an article
+  * @param {object} req - exporess request object
+  * @param {object} res - express response object
+  * @param {object} next - express next object
+  * @returns {object} returns an object of comment
+  */
+  static async deleteComment(req, res, next) {
+    const userId = req.userData.id;
+    const { commentId } = req.params;
+
+    let comment;
+    try {
```

Comment:
you do not need to add this query logic here: the `validateResourceId`  middleware automatically attaches the comment resource to the req object. Look up for this  `req.commentData`